### ISSUE TITLE
odyssey: VCF keyboard tracking has a rather drastic effect

### ISSUE THREAD

**Issue by nomadbyte** [nomadbyte/bristol-fixes#17]
ARP Odyssey: VCF slider in keyboard tracking mode (KBD CV switch up) seems to be overly aggressive??, non gradual. It jumps from bright sound at 0-level, but when off-zero it drops to very muffled sound, even at higher VCF cut-off.

The VCF slider is supposed to track the VCF cut-off freq, based on the selected voltage-source. In the KBD CV mode, the control voltages applied to cut-off should correspond to the pressed keys. I understand that VCF cut-off freq tracking is supposed to act similarly to keyboard tracking of the main OSC1,2 frequency (+/- 1 V/oct ). So raising the VCF tracking slider in KBD CV mode should result in a gradual raising of the VCF cut-off (sound brightens) up until the full level corresponding to the pressed key tuning level.

Basically, there should not be that drastic cut-off jump off-zero slider level.
